src/details/ArborX_AccessTraits.hpp

```diff
@@ -199,16 +199,18 @@ void check_valid_access_traits(PrimitivesTag, Primitives const &,
   }
 }
 
-template <typename Values>
+template <typename Values, typename Tag = PrimitivesTag>
```

Comment:
I thought about it. The thing is that in this case it would increase of size of this patch, as I would also have to change all places where I used `AccessValues` for primitives. I am not sure I want to do this.

Reply:
As long as it's a separate commit, it's pretty clean/clear.